### ISSUE TITLE
Support NIV

### ISSUE THREAD

**Issue by gerv** [biybl/biybl#33]
Our church uses the NIV, and the original version of BIYBL supported it. The new version does not, because it's not listed in @davidfedor's list of versions he got from the Digital Bible Platform API.

If Digital Bible Platform now supports NIV (unlikely?) we should switch to or add that option, but otherwise we may want to add an option to use a different API or even do HTML scraping.

**Comment by davidfedor** [comment]
No, the Digital Bible Platform doesn't support the NIV. It has these in English at the moment:
  Contemporary English Version
  English Standard Version   
  English Version for the Deaf
  King James Version
  NAS
  NLV
  World English Bible
(This list was obtained by parsing the json returned from http://dbt.io/library/volume?v=2&key=OURKEY&language_code=ENG

I think the only free NIV api is here: https://bibles.org/pages/api - looks like it will give what BIYBL will need though.

**Comment by gerv** [comment]
Adding another API to BIYBL is of course possible, but would be non-trivial. @cogiva: what do you think?
